### DISCUSSION TITLE
How to append value to child nested fields when calling append on parent component.

### DISCUSSION THREAD

**Discussion by Zhencha0Cai** [react-hook-form/react-hook-form#2598] [Q&A] [answered]
Hi,  my use case is very similar to the example of array of array fields. (https://codesandbox.io/s/react-hook-form-usefieldarray-nested-arrays-forked-xkncz?file=/src/fieldArray.js) The difference is that I would like to append the nested/child fields as well when I click the append button on the parent component. 
Here is the code for my attempt.
https://codesandbox.io/s/nervous-williamson-6i3lq?file=/src/App.js:741-745

Also I am getting fake path from the react-bootstrap's Form.File component, how can I resolve this? 

Appreciate any help.

**Comment by bluebill1049** [comment, marked as answer]
you can probably host your child useFieldArray at the parent level, then you can access the child level methods?

**Reply by Zhencha0Cai** [reply]
Thanks @bluebill1049 , I manage to do it by adding useEffect hook in child component. So every time the parent append function invoked, the child component will add the empty value.

**Reply by bluebill1049** [reply]
Nice 💯
